src/include/network/routines.rb

```diff
@@ -742,24 +756,62 @@ def SetIfaceDown(dev_name)
 
     # Tries to set all interfaces up
     #
-    # @return [boolean] false if some of interfaces cannot be set up
+    # @return [boolean] false if interfaces are empty or some
+    # interface cannot be bring up
```

Comment:
I just tried to comment what the API was doing in case of no interfaces, that could be unexpected as for example @ancorgs expected `true` instead of false if there was no interfaces to bring up. 

And we don't want to change the API at least for the SP2 release but we could consider it in master.